src/sonic-device-data/src/Makefile

```diff
@@ -0,0 +1,12 @@
+.ONESHELL:
+SHELL = /bin/bash
+.SHELLFLAGS += -e
+
+test:
+	# Execute Broadcom config file test
+	pushd ../tests/
+	for f in `find ../../../device -name "*.config.bcm"`; do
```

Comment:
Suggest replacing deprecated backticks with `$()` syntax: `for f in $(find ../../../device -name "*.config.bcm"); do`